content/topics/Analyze/machine-learning/ml-intro/machine-learning-101.md

```diff
@@ -0,0 +1,141 @@
+date: 26/02/2024
+weight: #1
+author: "Lisa Holling"
+authorlink: "https://tilburgsciencehub.com/contributors/lisaholling/"
+aliases:
+  - /machinelearning
+  - /supervised
+  - /unsupervised
+  - /reinforcement
+
+---
+
+## Overview
+Are you intrigued by the concept of machine learning but find yourself daunted by complex theories and techniques?  No need to worry - this article has got you covered. It explains the basics of machine learning and guides you in setting up your own machine modeling project. 
+
+The article covers several topics:
+1. What is Machine Learning?
+2. Types of Machine Learning
+4. Machine Learning Algorithm examples 
+5. Tips for Setting Up Your Machine Learning Project
+
+
+
+## What is Machine Learning? 
+At its core, machine learning is a branch of artificial intelligence that empowers systems to learn and improve from experience without being explicitly programmed. These systems use data to identify patterns, make decisions, and improve their performance over time. 
+
+Typically, the goal of machine learning is to develop algorithms that can learn how to perform tasks based on input data. For example, a machine learning model might be trained to predict sales or forecast customer churn based on historical data. Here's a visualization to illustrate the concept of a basic machine learning model.
+
+<p align = "center">
+<img src = "../images/basic-ml-visualization.png" width="700">
+<figcaption> Visualization of a basic machine learning model, image by the author </figcaption>
+</p>
+
+
+## Types of Machine Learning
+Machine learning can be categorized into several types, each designed for different data types and objectives. 
+Traditional machine learning (including supervised and unsupervised learning) splits data into a training and testing set to identify patterns and apply them to new data through predictions. Reinforcement learning, on the other hand, does not rely on a predefined dataset but interacts with an environment to learn.
+
+
+
+### 1. **Supervised Learning** 
+<!-- link to supervised ml bb -->
+
+Supervised learning trains an algorithm on a labeled dataset, where both the input data and the desired output are provided. The model learns to map inputs to outputs, ideal for making predictions on unseen data.
+
+Supervised learning uses information from a training set to predict a given outcome variable. An algorithm is trained on a labeled dataset, which means that both the input data and the desired output are given. The model learns to map inputs to outputs, ideal for making predictions on unseen data. 
+
+Different approaches are **classification** and **regression**: 
+* Classification predicts categorical outcomes, such as labeling students' course feedback as positive or negative. 
+* Regression predicts continuous outcomes, like forecasting sales of a newly launched product.
+
+Popular supervised learning algorithms include linear regression, logistic regression, and decision trees.
```

Comment:
Instead of just popular I'd say *'some of the most popular'* as for instance KNN or SVM are also very popular in traditional supervised ML.